account/account_test.go

```diff
@@ -478,10 +480,10 @@ func TestAddInvoke(t *testing.T) {
 		"devnet": {},
 		"testnet": {
 			{
-				// https://goerli.voyager.online/tx/0x73cf79c4bfa0c7a41f473c07e1be5ac25faa7c2fdf9edcbd12c1438f40f13d8#overview
+				// https://sepolia.voyager.online/tx/0x5d307ad21a407ab6e93754b2fca71dd2d3b28313f6e844a7f3ecc404263a406#overview
```

Comment:
The testdata from these cases needs to come from the Sepolia network. I would suggest creating it using a test wallet

Reply:
Looks like is everything good with these transactions.